router/router.go

```diff
@@ -125,7 +125,7 @@ func Load(options ...gin.HandlerFunc) *gin.Engine {
 		ScmHandlers(baseAPI)
 
 		// Search endpoints
-		SearchHandlers(baseAPI)
```

Comment:
Was removing this an accident or on purpose?
It looks like `SearchHandlers` is now unused on main. So, if `/api/v1/dashboards` is actually supposed to replace `/api/v1/search/builds`, then that function needs to also be removed. Otherwise, we probably need to re-add the `SearchHandlers` call here.